platform/x11/os_x11.cpp

```diff
@@ -215,7 +214,7 @@ void OS_X11::initialize(const VideoMode &p_desired, int p_video_driver, int p_au
 			XIFreeDeviceInfo(info);
 
 			if (!touch.devices.size()) {
-				fprintf(stderr, "No suitable touch device found\n");
+				fprintf(stderr, "No touch devices found\n");
```

Comment:
This is quite verbose btw. Godot is sadly not so well equipped to handle proper info logging that could be toggled via switch, but you could put this behind the verbose mode flag.

Reply:
Also, remember that you can use `ERR_PRINT`, `ERR_PRINTS`, etc. when relevant (not suitable for this one as it's not an actual error, but for the above ones I guess it may be).

Reply:
Since I plan to revisit this very soon, I'll improve all that.